models/Clients.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const ClientsSchema = new Schema({
+  name: { type: String, required: true },
+  phone: { type: Number, required: true },
+  location: {
+    country: { type: String, required: true },
+    state: { type: String, required: true },
+    city: { type: String, required: true },
+    address: { type: String, required: true },
+  },
+  logo: String,
+  description: String,
+  created: {
+    admin: { type: mongoose.Schema.Types.ObjectId, ref: 'Admins' },
+  },
+  modified: {
```

Comment:
For this version, I do not think it is necessary to have a record of who created or changed each client.

Reply:
Okey, maybe we dont need it now.